src/pages/index.tsx

```diff
@@ -5,6 +5,12 @@ const HomePage = () => {
     <div>
       <h1>This is Code review tester app</h1>
       <p>This is a subtitle</p>
+      <ul>
+        <li>Item 1</li>
+        <li>Item 2</li>
+        <li>Item 3</li>
+        <li>Item 4</li>
+      </ul>
```

Comment:
**suggestion (performance):** Consider performance implications of frequent re-renders

The use of useDyteSelector for active participants might cause frequent re-renders. Consider using useMemo or a custom hook to memoize this value and prevent unnecessary re-renders of child components.

```suggestion
const activeParticipants = useMemo(() => {
  return useDyteSelector((meeting) => meeting.participants.active.toArray());
}, []);
```